src/umd/shims/ReactAddonsDOMDependenciesUMDShim.js

```diff
@@ -9,24 +9,22 @@
  * @providesModule ReactAddonsDOMDependenciesUMDShim
  */
 
-/* globals ReactDOM */
-
 'use strict';
 
-exports.getReactDOM = function() {
-  return ReactDOM;
-};
+function getReactDOM() {
+  var ReactWithAddonsUMDEntry = require('ReactWithAddonsUMDEntry');
```

Comment:
This should be safe since we only use `ReactAddonsDOMDependenciesUMDShim` in the addons build. Again, build sizes looked normal.